### DISCUSSION TITLE
Site Ownership with Remote_User_Guard

### DISCUSSION THREAD

**Discussion by BlueDevil08** [firefly-iii/firefly-iii#5295] [Firefly III]
Hi All,  

Just checking out Firefly and used the AUTHENTICATION_GUARD environment variable to allow Authelia to handle authentication.  The issue I've stumbled on is that I'm unable to enable/disable currencies and it see a notification "Message: Please ask SITE_OWNER to add, remove or edit currencies." I have tried putting the username and also the Firefly User Id (1) that I want to be the Site Owner in that environment variable, but it does not seem to give it the admin rights.  Looking for a little help here as I'm sure this is a simple issue and I'm just missing something.

Thanks!

**Comment by JC5** [comment]
The first user that's registered in Firefly III gets the site owner role. If the auth guard is set up later or at a different time, Firefly III may have created a second user that won't be given site owner.

Go to `/profile` and write down your user ID. Then, in the database, add a new entry to the `role_user` table. For the `user_id` value, fill in your user ID. For the `role_id` value, fill in `1`.

**Reply by nottheend** [reply]
Thanks @JC5 
That helps me with the pain of not knowing who the admin is.

After enabling currencies which I don't use I can't disable them since I didn't remember the Admin anymore😂

Thanks!

@BlueDevil08 Did you consider to mark this discussion as 'answered'?

**Reply by JC5** [reply]
Hah, I still had to point you to this answer, my apologies for making you wait.
